benchmarks/c/miscellaneous/memcpy_int.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+#include <assert.h>
+#include <string.h>
+
+int src[5] = {0,1,2,3,4};
+int dest[5];
+
+int main () {
+
+#ifdef FAIL
+    int bytesToCopy = sizeof(src) - sizeof(int);
+#else
+    int bytesToCopy = sizeof(src);
+#endif
+    memcpy(dest, src, bytesToCopy);
+    for(int i = 0; i < sizeof(src); i++) {
+        assert(dest[i] == src[i]);
+    }
```

Comment:
Is this even correct? I think `sizeof(src)` returns `20` rather than `5`. So this program has UB.
I think it should be `sizeof(src) / sizeof(src[0])` to get the number of indices (or just use `#define size 5`);